action/penalization/release.go

```diff
@@ -0,0 +1,122 @@
+package penalization
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+	"github.com/tendermint/tendermint/libs/kv"
+
+	"github.com/Oneledger/protocol/action"
+	gov "github.com/Oneledger/protocol/data/governance"
+	"github.com/Oneledger/protocol/data/keys"
+)
+
+var _ action.Msg = &Release{}
+
+type Release struct {
+	ValidatorAddress keys.Address
+}
+
+func (r Release) Marshal() ([]byte, error) {
+	return json.Marshal(r)
+}
+
+func (r *Release) Unmarshal(data []byte) error {
+	return json.Unmarshal(data, r)
+}
+
+func (r Release) Signers() []action.Address {
+	return []action.Address{r.ValidatorAddress.Bytes()}
+}
+
+func (r Release) Type() action.Type {
+	return action.RELEASE
+}
+
+func (r Release) Tags() kv.Pairs {
+	tags := make([]kv.Pair, 0)
+
+	tag := kv.Pair{
+		Key:   []byte("tx.type"),
+		Value: []byte(r.Type().String()),
+	}
+	tag2 := kv.Pair{
+		Key:   []byte("tx.validator"),
+		Value: r.ValidatorAddress.Bytes(),
+	}
+
+	tags = append(tags, tag, tag2)
+	return tags
+}
+
+var _ action.Tx = releaseTx{}
+
+type releaseTx struct{}
+
+func (rtx releaseTx) Validate(ctx *action.Context, tx action.SignedTx) (bool, error) {
+	r := &Release{}
+	err := r.Unmarshal(tx.Data)
+	if err != nil {
+		return false, errors.Wrap(action.ErrWrongTxType, err.Error())
+	}
+	err = action.ValidateBasic(tx.RawBytes(), r.Signers(), tx.Signatures)
+	if err != nil {
+		return false, err
+	}
+
+	feeOpt, err := ctx.GovernanceStore.GetFeeOption()
```

Comment:
We should access fee options through the context object instead ,ctx.FeePool.GetOpt()